nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -464,7 +464,7 @@ in
       };
 
       stateDir = mkOption {
-        default = "/var/spool/nginx";
+        default = null;
         description = "
           Directory holding all state for nginx to run.
         ";
```

Comment:
This option should be removed with `mkRemovedOptionModule`, instead of leftover like this.

Reply:
Fixed, thanks.